src/components/card/card.ts

```diff
@@ -0,0 +1,73 @@
+import {Component, View, ViewEncapsulation} from 'angular2/core';
+import {CONST_EXPR, Type} from 'angular2/src/facade/lang';
+
+/*
+
+<md-card> is a basic content container component that adds the styles of a material design card.
+
+While you can use the tags alone, it also provides a number of preset styles for common card sections, including:
+ - md-card-title
+ - md-card-subtitle
+ - md-card-content
+ - md-card-actions
+ - md-card-footer
+
+ You can see some examples of cards here:
+ http://embed.plnkr.co/s5O4YcyvbLhIApSrIhtj/
+
+*/
+
+@Component({
+  selector: 'md-card',
+  templateUrl: './components/card/card.html',
+  styleUrls: ['./components/card/card.css'],
+  encapsulation: ViewEncapsulation.None
+})
+export class MdCard {}
+
+
+/*********************************************************************************************************
```

Comment:
I'm not a fan of section-delimiter comments; two blank lines between the components should provide enough visual separation.